### PR TITLE
[CARBONDATA-3126] Correct some spell error in CarbonData

### PR DESCRIPTION
**Pull request by tisonkong**
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed?
 some spell error，modified some local variable quantity.
 - [x] Any backward compatibility impacted?
 No
 - [x] Document update required?
No
 - [x] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.

### CODE REVIEW

File: core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java [hunk 1]
```diff
@@ -84,7 +84,7 @@
   private static final CarbonProperties CARBONPROPERTIESINSTANCE = new CarbonProperties();
 
   /**
-   * porpeties .
+   * porpeties
```
Comment (xubo245) [inline core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java]: porpeties should be Properties, can you describe it in detail?

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by xubo245** [comment]
add to whitelist

**Comment by chenliang613** [comment]
add to whitelist

**Comment by chenliang613** [comment]
LGTM.
One small issue, for pr title, please add one "blank" after [CARBONDATA-3126]

**Comment by tisonkong** [comment]
> LGTM.
> One small issue, for pr title, please add one "blank" after [CARBONDATA-3126]

No problem. I see some has blank and some not, i will add blank after brackets next time .

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2077/

**Comment by CarbonDataQA** [comment]
Build Failed  with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10331/

**Comment by CarbonDataQA** [comment]
Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2282/

File: core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java (hunk 1 above)
Comment (tisonkong) [reply core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java]: Right, this spell error has corrected

**Comment by xubo245** [comment]
Please fix the CI error.

**Comment by tisonkong** [comment]
> Please fix the CI error. @tisonkong

I found the error reason is "line is longer than 100 characters" in DictionaryBasedVectorResultCollector. I have take a newline to fix this problem.

**Comment by tisonkong** [comment]
> Build Failed with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2282/

I found the error reason is "line is longer than 100 characters" in DictionaryBasedVectorResultCollector. I have take a newline to fix this problem. Please try agin, thank u.

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder2.1/2081/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.2.1, Please check CI http://95.216.28.178:8080/job/ApacheCarbonPRBuilder1/2286/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.3.2, Please check CI http://136.243.101.176:8080/job/carbondataprbuilder2.3/10335/

**Comment by xubo245** [comment]
LGTM，Thank you for working on it. @tisonkong

**Pull request closed without merging**